examples/shell.py

```diff
@@ -12,12 +12,12 @@ async def run():
     drone = System()
     await drone.connect(system_address="udpin://0.0.0.0:14540")
 
-    tasks = {asyncio.create_task(observe_shell(drone))}
+    _tasks = {asyncio.create_task(observe_shell(drone))}
```

Comment:
It is unusual to assemble these tasks but not await them.  Are we certain that they will be run at some point?